run.py

```diff
@@ -1,50 +1,66 @@
-import asyncio
+import os
 import random
-import re
 import string
+import asyncio
+from collections import namedtuple
 
 import discord
 
-from cybot import client
+from plugins import *
+from cybot import client, utils
 from cybot.settings import (
     TOKEN,
     user_kick_timeout,
-    eng, heb
+    eng, heb,
+    CMD_SIGN
 )
 
-from plugins import commands, async_commands, karma_store_cmds
-from plugins.karma import _take_karma
+
+MemberOnServer = namedtuple('MemberOnServer', 'user server')
+Command = namedtuple('Command', 'function channels')
 
 unverified = {}
+commands = {}
```

Comment:
why it's located here?

Reply:
for the process_cmd func

Reply:
I think it's will be better to use `cybot` package for these kind of vars

Reply:
but its var that used only on process_cmd and the loop at the outer scope that inserting all the commands from the register_command in the right format so process_cmd could use it